packages/govern-server/src/core/params.ts

```diff
@@ -0,0 +1,17 @@
+export const NETWORKS = [
```

Comment:
This could probably be within a ``config.json`` file. I think it would also be more self-explaining instead of calling this file ``params``.